docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -254,7 +261,7 @@
                 </tr>
 
                 <tr>
-                    <td class="property" id="job-systemstreampartition-matcher-class">job.systemstreampartition.matcher.class</td>
+                    <td class="property" id="job-systemstreampartition-matcher-class">job.systemstreampartition.<br>matcher.class</td>
```

Comment:
It was a convention for readability that was abandoned. Since the property and default columns are set to no-wrap, large values increase the table size and push the description off the page. When you see the result, you'll like it. :-)